lua/lvim/plugins.lua

```diff
@@ -29,10 +29,10 @@ local commit = {
   popup = "b7404d35d5d3548a82149238289fa71f7f6de4ac",
   project = "71d0e23dcfc43cfd6bb2a97dc5a7de1ab47a6538",
   structlog = "6f1403a192791ff1fa7ac845a73de9e860f781f1",
-  telescope = "9aaaa0c5f3eb665b51bbcafda084de4b0952fef0",
+  telescope = "88437804e157196f053d0fa62dc891facd9ab746",
   telescope_fzf_native = "b8662b076175e75e6497c59f3e2799b879d7b954",
   toggleterm = "265bbff68fbb8b2a5fb011272ec469850254ec9f",
-  which_key = "0fd9de78fe09215e1b7c6173ff1b0b90c8ed6ec4",
+  which_key = "312c386ee0eafc925c27869d2be9c11ebdb807eb",
```

Comment:
@kylo252 this is causing errors
<img width="1353" alt="Screen Shot 2021-12-28 at 12 23 24 PM" src="https://user-images.githubusercontent.com/10992695/147548053-1ebc6b75-1ce6-45b1-b9b1-cf384b391e0d.png">

Reply:
nvm you can fix the which-key error by doing a `LvimCacheReset`